web-server/src/controllers/tasks/edit.ts

```diff
@@ -0,0 +1,50 @@
+import { Request, Response, NextFunction } from 'express';
+
+import { AppDataSource } from 'orm/data-source';
+import { Task } from 'orm/entities/tasks/Task';
+import { CustomError } from 'utils/response/custom-error/CustomError';
+
+export const edit = async (req: Request, res: Response, next: NextFunction) => {
+  const id = parseInt(req.params.id);
+  const { title, slug, description, statement } = req.body;
```

Comment:
Can we also edit the rest of the fields of the `Task` ? There was a lot of those including `checker` and other stuff important for the problem setter.